### PR TITLE
ROX-25330: Adding enforcement to the policy violations filter

### PR DESCRIPTION
**Pull request by sachaudh**
### Description

This PR adds a multi-select dropdown for the Enforcement action filter in Policy Violations.

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing

- [x] inspected CI results

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests
- [x] no changes

#### How I validated my change

Screenshot 2024-08-07 at 7 30 27 AM

### CODE REVIEW

File: ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts [hunk 1]
```diff
@@ -16,4 +16,32 @@ export const Category: CompoundSearchFilterAttribute = {
...
+            { value: 'FAIL_BUILD_ENFORCEMENT', label: 'Fail build enforcement' },
+            { value: 'FAIL_KUBE_REQUEST_ENFORCEMENT', label: 'Fail kube request enforcement' },
+            {
+                value: 'FAIL_DEPLOYMENT_CREATE_ENFORCEMENT',
+                label: 'Fail deployment create enforcement',
+            },
+            {
+                value: 'FAIL_DEPLOYMENT_UPDATE_ENFORCEMENT',
+                label: 'Fail deployment update enforcement',
+            },
```
Comment (dvail) [inline ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts:42]: Nits: 
- could we remove `"enforcement"` from the end of each label, it seems redundant with the other text?
- could "Unset" be the last option in the list?

Comment (dvail) [reply ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts:42]: Or - could the options be alphabetical? In that case "unset" would naturally fall to the bottom.

**Review by dvail — APPROVED** [review]
A few readability nits, but otherwise LGTM

File: ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Components/CompoundSearchFilter/attributes/policy.ts:42]: Sure, we can do that 👍🏼

**Pull request merged**